multi-agent/lib/mas/elements/nodes/deep_agent/deep_agent_node.py

```
@@ -89,87 +92,166 @@ def __init__(
+
         self._ensure_compiled()
-        self.process_packets(state)
+        try:
+            self.process_packets(state)
+        finally:
+            close_tools(self._domain_tools)
+            if self._sandbox is not None:
+                try:
+                    self._sandbox.close()
+                except Exception:
+                    pass
```

Ruff (0.15.20)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)